powerline/bindings/bar/powerline-bar.py

```diff
@@ -2,31 +2,34 @@
 # vim:fileencoding=utf-8:noet
 from __future__ import (unicode_literals, division, absolute_import, print_function)
 
+import os
 import sys
 import time
 
 from threading import Lock, Timer
 from argparse import ArgumentParser
 
-from powerline import Powerline
+from powerline.lemonbar import LemonbarPowerline
 from powerline.lib.encoding import get_unicode_writer
 
 
-class BarPowerline(Powerline):
-	get_encoding = staticmethod(lambda: 'utf-8')
-
-	def init(self):
-		super(BarPowerline, self).init(ext='wm', renderer_module='bar')
-
-
 if __name__ == '__main__':
-	parser = ArgumentParser(description='Powerline BAR bindings.')
+	parser = ArgumentParser(description='Powerline lemonbar bindings.')
 	parser.add_argument(
 		'--i3', action='store_true',
 		help='Subscribe for i3 events.'
 	)
+	parser.add_argument(
+		'--width', action='store', type=int,
+		default=None, help='Maximum width the bar can occupy'
+	)
+	parser.add_argument(
+		'theme', nargs='?', default='default',
+		help='The theme to render')
```

Comment:
My argument regarding overrides applies here as well (also I do not see what is the reasoning behind adding functionality to deprecated file).